tests/vmi_multus_test.go

```diff
@@ -327,20 +333,21 @@ var _ = Describe("Multus", func() {
 				tests.WaitUntilVMIReady(vmiOne, tests.LoggedInAlpineExpecter)
 				tests.WaitUntilVMIReady(vmiTwo, tests.LoggedInAlpineExpecter)
 
-				configInterface(vmiOne, "eth0", "10.1.1.1/24", "localhost:~#")
+				configInterface(vmiOne, "eth0", vmiOneCIDR, "localhost:~#")
 				By("checking virtual machine interface eth0 state")
 				checkInterface(vmiOne, "eth0", "localhost:~#")
 
-				configInterface(vmiTwo, "eth0", "10.1.1.2/24", "localhost:~#")
+				configInterface(vmiTwo, "eth0", vmiTwoCIDR, "localhost:~#")
 				By("checking virtual machine interface eth0 state")
 				checkInterface(vmiTwo, "eth0", "localhost:~#")
 
 				By("ping between virtual machines")
-				pingVirtualMachine(vmiOne, "10.1.1.2", "localhost:~#")
+				err := testPingFromVMI(vmiOne, cidrToIP(vmiTwoCIDR), "localhost:~#")
+				Expect(err).ToNot(HaveOccurred(), "VMI %s  and VMI %s  should reach each other", vmiOne.Name, vmiTwo.Name)
```

Comment:
You not pinging to each other, you ping from `vmiOne` to `vmiTwo`.
I would change to `"VMI %s  ne able to ping  VMI %s ", vmiOne.Name, vmiTwo.Name`